src/main/java/mate/jdbc/services/impl/ManufacturerService.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.services.impl;
+
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.services.GenericService;
+
+@Service
+public class ManufacturerService implements GenericService<Manufacturer> {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        return manufacturerDao.get(id).orElseThrow(() ->
+                new RuntimeException("No object in DB with id=" + id));

```

Comment:
```suggestion
                new DataNotFoundExeption("No object in DB with id=" + id));
```
better use custom exception here